azure-functions/src/functions/HttpExample.js

```diff
@@ -1,3 +1,4 @@
+const newrelic = require('newrelic');
```

Comment:
I'm unsure of the value of requiring `newrelic` in the file when we're not using `newrelic` methods.